optimum/intel/openvino/modeling_decoder.py

```diff
@@ -950,6 +947,47 @@ def _from_pretrained(
         return causal_model
 
 
+class OVPhi3ForCausalLM(OVModelForCausalLM):
+    def prepare_inputs_for_generation(
```

Comment:
would you mind adding a link to the original code 
https://github.com/huggingface/transformers/blob/v4.57.0/src/transformers/models/phi3/modeling_phi3.py#L493

Reply:
Done